.cirrus.yml

```diff
@@ -795,7 +798,7 @@ podman_machine_mac_task:
     skip: *skip_rhel_release
     depends_on: *build
     persistent_worker: *mac_pw
-    timeout_in: 45m
+    timeout_in: 60m
```

Comment:
Excellent. Maybe hit Windows as well? I see timeouts there occasionally

Reply:
yeah I am mean it is totally the wrong thing to do because the tests must be faster...
But if it avoids flakes it is still faster than rerunning I don't think I have seen windows timeouts in a long time so I didn't add that. If you say that times out as well I can add it for sure.

Reply:
this is a good change .. if we touch this again, should we make it a variable ?